Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -922,7 +910,6 @@ class SmartPage_Context
         message +
         '</td></tr></table>';
       statusMessageWindow.appendChild (statusMessageBlock);
-      $(statusMessageBlock).iFrameShim({ top: '0px', left: '0px', width: '100%', height: '100%' });
```

Comment:
Do we still have usages of iFrameShim? Need to remember to update or delete iFrameShim when we do jQuery noConflict.

Reply:
Only in jquery.js so we should be fine